draft-ietf-quic-transport.md

```diff
@@ -73,9 +73,11 @@ informative:
     title: "Structured Streams: A New Transport Abstraction"
     author:
       - ins: B. Ford
-    date: 2007-08
+    date: 2007-10
```

Comment:
This is odd. This depends on how you write the ref. SIGCOMM was in Aug 2007, the current citation could say Proceedings of ACM SIGCOMM, August 2007.  ACM CCR carries all SIGCOMM publications in the next issue, as a special issue, which must have come out in Oct 2007. I'd keep the Proceedings date, since that's what's usually used.